Code/Framework/AzToolsFramework/AzToolsFramework/Metadata/MetadataManager.h

```diff
@@ -90,7 +90,7 @@ namespace AzToolsFramework
 
     public:
         bool GetValue(AZ::IO::PathView file, AZStd::string_view key, void* outValue, AZ::Uuid typeId) override;
-        bool GetJsonValue(AZ::IO::PathView file, AZStd::string_view key, rapidjson_ly::Value& outValue) override;
+        bool GetJsonValue(AZ::IO::PathView file, AZStd::string_view key, rapidjson_ly::Document& outValue) override;
```

Comment:
Is this method misnamed now? It seems to make sense to pass a `Value&` out-parameter to `GetJsonValue()`, but less so to pass a `Document&`. But maybe the purpose of the method didn't change, so the name still is appropriate?

Reply:
Maybe its misnamed.  Ideally we'd potentially start moving to AZ::Dom, in which case, Value won't matter.  in rapidjson, Document is a value, because it derives from value, but it gets weird...

Reply:
How about just GetJson?